lms/djangoapps/courseware/courses.py

```diff
@@ -294,6 +294,13 @@ def get_course_info_section(request, user, course, section_key):
     return html
 
 
+def get_course_upgrade_link(course, user):
```

Comment:
Nit: why not just reference VerificationDeadlineDate(course, user).link directly where this method is called, instead of creating a wrapper around it? VerificationDeadlineDate is not private to (or even defined in) this file.